lms/static/styles/components/_SvgIcon.scss

```diff
@@ -3,7 +3,6 @@
   display: flex;
 
   &--inline {
-    
```

Comment:
I don't much mind removing a line break here, but wanted to be sure you intended this change.

Reply:
Yes, this just came from copying over `svg-icon.scss` from the client.